src/test/java/healthy/lifestyle/backend/users/controller/UserControllerTest.java

```diff
@@ -173,6 +172,224 @@ void updateUserTest_shouldReturnErrorMessageAnd400_whenUserResourceMismatch() th
                 .andDo(print());
     }
 
+    @Test
+    @WithMockUser(
+            username = "username-one",
+            password = "password-one",
+            authorities = {"ROLE_USER"})
+    void updateUserTest_shouldReturnErrorMessageAnd400_whenUsernameHasInvalidSymbol() throws Exception {
+        // Given
+        Role role = dataHelper.createRole("ROLE_USER");
+        Country country = dataHelper.createCountry(1);
+        Integer age = 20;
+        User user = dataHelper.createUser("two", role, country, null, age);
+        Country updatedCountry = dataHelper.createCountry(2);
+        UpdateUserRequestDto requestDto = dataHelper.createUpdateUserRequestDto("two", updatedCountry.getId(), 35);
+        requestDto.setUpdatedUsername("two2/&%*)_-=+!@");
+
+        String REQUEST_URL = URL + "/{userId}";
+
+        // When
+        mockMvc.perform(patch(REQUEST_URL, user.getId())
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(requestDto)))
+                // Then
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.updatedUsername", is("Not allowed symbols")))
+                .andDo(print());
+    }
+
+    @Test
+    @WithMockUser(
+            username = "username-one",
+            password = "password-one",
+            authorities = {"ROLE_USER"})
+    void updateUserTest_shouldReturnErrorMessageAnd400_whenUsernameOutOfSize() throws Exception {
+        // Given
+        Role role = dataHelper.createRole("ROLE_USER");
+        Country country = dataHelper.createCountry(1);
+        Integer age = 20;
+        User user = dataHelper.createUser("two", role, country, null, age);
+        Country updatedCountry = dataHelper.createCountry(2);
+        UpdateUserRequestDto requestDto = dataHelper.createUpdateUserRequestDto("two", updatedCountry.getId(), 35);
+        requestDto.setUpdatedUsername("two");
+
+        String REQUEST_URL = URL + "/{userId}";
+
+        // When
+        mockMvc.perform(patch(REQUEST_URL, user.getId())
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(requestDto)))
+                // Then
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.updatedUsername", is("size must be between 6 and 20")))
+                .andDo(print());
+    }
+
+    @Test
+    @WithMockUser(
+            username = "username-one",
+            password = "password-one",
+            authorities = {"ROLE_USER"})
+    void updateUserTest_shouldReturnErrorMessageAnd400_whenEmailHasInvalidSymbol() throws Exception {
+        // Given
```

Comment:
This test is failed.

![image](https://github.com/Healthy-Lifestyle-Assistant/backend/assets/93421769/47e97b7d-8fe0-4e9e-8626-b8b6664974fb)

But sometimes it works successfully.